dbms/src/Core/Field.cpp

```diff
@@ -69,6 +69,13 @@ namespace DB
                     x.push_back(value);
                     break;
                 }
+                case Field::Types::Uuid:
```

Comment:
Looks like Field is generic enough to have type be named UInt128 instead of UUID.
(The case: what if anyone would like to introduce another type for IPv6?)